plugins/a11yhelp/plugin.js

```diff
@@ -60,21 +63,27 @@
 	 */
 	CKEDITOR.plugins.a11yhelp = {
 		/**
-		 * Return html which represent keystroke. Result is wrapped with `<kbd></kbd>` tags.
+		 * Property contains current key map list used by {@link CKEDITOR.plugins.a11yhelp#representKeystroke}.
+		 * After setting up with {@link CKEDITOR.plugins.a11yhelp#_renderKeyMap}. Property cab be read directly.
 		 *
-		 * 		CKEDITOR.plugins.a11yhelp.representKeystroke( CKEDITOR.instances.editor, 4456496 );
-		 * 		"<kbd><kbd>Alt</kbd>+<kbd>0</kbd></kbd>"
+		 * @since 4.8.0
+		 * @private
+		 * @property {Object}
+		 */
+		_keyMap: undefined,
+
+		/**
+		 * Method render and return {@link CKEDITOR.plugins.a11yhelp#_keyMap} property.
 		 *
 		 * @since 4.8.0
-		 * @param {CKEDITOR.editor} editor Editor instance.
-		 * @param {Number} keystroke Number which represent command keystroke.
-		 * @returns {String} HTML code with key names wrapped with `<kbd></kbd>` tags.
+		 * @private
+		 * @param {CKEDITOR.editor} editor Editor instance
+		 * @returns {Object}
 		 */
-		representKeystroke: function( editor, keystroke ) {
+		_renderKeyMap: function( editor ) {
```

Comment:
I think we can simplify creating `keyMap` even more, just creating singleton like method:

```
_renderKeyMap: ( function() {
    var keyMapCache;
    return function( editor ) {
        if ( !keyMapCache ) {
            // Here goes the body of the current function. Created keyMap should be assigned to keyMapCache variable.
        }
        return keyMapCache;
    }
} )();
```

And then we can completely drop `_keyMap` property and just use `var keyMap = this._renderKeyMap( editor );` in other places.

---

Also `renderKeyMap` is somehow misleading name, this functions does not render anything it creates/generates key map object so it can be e.g. `[create/generate/get]keyMap`.